en/services/high_latency.md

```diff
@@ -10,18 +10,18 @@ The protocol provides a heartbeat-like message ([HIGH_LATENCY2](#HIGH_LATENCY2))
 High latency links are expensive.
 In order to reduce traffic to the bare minimum, some of the fundamental assumptions of MAVLink are explicitly broken:
 - [HEARTBEAT](../messages/common.md#HEARTBEAT) messages are not emitted on the channel (either by the autopilot or GCS).
-
-  > **Note** The heartbeat is used to build MAVLInk routing tables between channels.
+  > **Note** The heartbeat is used to build MAVLink routing tables between channels.
     Commands addressed specifically to the high latency component may not be routed from another channel (i.e. you can connect to the component from a GCS directly, but not via a MAVLink router).
-- Broadcast messages from the network should not be automatically sent over the high latency channel.
+- Only the [COMMAND](./command) service messages should be sent over the high latency channel.
+- Ground stations should not send or receive [MISSION](./mission) items or [PARAMETER](./parameter) items over the high latency channel in order to prevent congestion.
 
 The other rules are essentially the same but there are some implications of the above changes:
-- Broadcast messages from the high latency channel should be routed to other nodes on the network as usual.
+- Messages from the high latency channel should be routed to other nodes on the network as usual.
   Note that this in reality most systems on a high latency network **only** send `HIGH_LATENCY2`.
-- Addressed messages should be sent over the high latency channel (in both directions) in accordance with the normal routing rules.
-  In practice the lack of `HEARTBEAT` means that addressed messages are unlikely to arrive, and hence be sent.
+- Most addressed messages will not be sent over the high latency link.
```

Comment:
I disagree with this change. An addressed message should always be sent, since it is intended for the target. What we're trying to say is that the GCS should not be sending ad hoc commands.

The only way we could change/should reasonably change this would be to say something like "the only messages that can be sent over the HL channel are the HL2 and the command protocol messages". In other words to disallow everything else.

I don't think that is what we want to do. It is a clear practical direction and it works for the main use case, but what it also does is mean that you literally can't do anything but observe the beacon. Is that what you think we want?

Reply:
Removed change. Added clarification about GCS managing the messages it sends to autopilot